### PR TITLE
feat(seventv): init

### PR DESCRIPTION
**Pull request by mxgic1337**
## 🎉 Theme for [7TV](https://7tv.app) 🎉

7TV is a platform and a browser extension that provides custom emotes for Twitch.
Preview

## 💬 Additional Comments 💬

This userstyle only themes the 7tv.app website. The extension is unthemed.

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - [x] `catppuccin.user.css` - all the CSS for the userstyle, based on the
        template.
  - [x] `preview.webp` - composite image of all four individual flavor screenshots (taken with the default accent color of mauve) stitched together, generated via [Catwalk](https://github.com/catppuccin/catwalk).

### CODE REVIEW

File: styles/seventv/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,390 @@
...
+    input,
+    textarea {
+      &::placeholder {
+        color: @subtext0 !important;
+      }
+    }
+
+    background-color: @base;
+
+    & {
```
Comment (isabelroses) [inline styles/seventv/catppuccin.user.css:82]: If we are just using `&` we can remove it
```suggestion
```

**Review by isabelroses — CHANGES_REQUESTED** [review]
LGTM overall, 1 small issue with the gradients

**Comment by mxgic1337** [comment]
@isabelroses What is the issue? Also, should I rename the file to `catppuccin.user.less`?

**Comment by isabelroses** [comment]
Oh the image didn't upload...
image

>  Also, should I rename the file to catppuccin.user.less?

Yes, you can also remove the `vim` line at the bottom of the file.

**Comment by mxgic1337** [comment]
I'm not sure how I should style these gradients.

**Comment by isabelroses** [comment]
> I'm not sure how I should style these gradients.

Thats pretty fair. I don't mind removing them since there is a few. Alternatively, you could reduce them a little. Like I choose herre

```less
radial-gradient(150% 180% at 50% 180%, #fab387, #f9e2af 1%, #cba6f7 25%, #181825 75%), var(--secondary) !important
```

cc: @uncenter

**Review by isabelroses — APPROVED** [review]

**Comment by uncenter** [comment]
This was not approved by both staff members, as is required. Only staff should be merging init pull requests.

**Comment by uncenter** [comment]
I'm going to leave review comments on this pull request hopefully later today that will need to be addressed in a follow up PR.

**Comment by mxgic1337** [comment]
> This was not approved by both staff members, as is required. Only staff should be merging init pull requests.

Oops, I thought that only one approval is required, sorry!

**Comment by uncenter** [comment]
No worries, we need to work on our documentation and maintainer flow! It's definitely not clear at the moment. See the new "Custom GitHub bot" section on https://github.com/catppuccin/userstyles/issues/1357 for what it might look like in the future :)

**Comment by uncenter** [comment]
> I'm going to leave review comments on this pull request hopefully later today that will need to be addressed in a follow up PR.

Looks really good! No changes from me, thanks for your work on this!

**Pull request merged**